Extensions/GoogleAnalyticsExperiments/Src/Tasks/GoogleAnalyticsExperiments/index.ts

```diff
@@ -0,0 +1,81 @@
+import {IExperiment} from './models/IAnalytics';
+import {TaskOperation} from './operations/TaskOperation';
+import {TaskParameter} from './models/TaskParameter';
+import * as tl from 'azure-pipelines-task-lib/task';
+import * as schema from './models/Schema.json';
+const path = require('path');
+const fs = require('fs');
+
+async function run() {
+    try {
+        tl.setResourcePath(path.join(__dirname, 'task.json'));
+        let param: TaskParameter = TaskParameter.getInstance();
+        let trafficCoverage: number | null = param.trafficCoverage;
+        let equalWeighting: boolean = param.equalWeighting;
+        let action: string = param.action;
+        let filePath: string = param.filePath;
+
+        let taskOperation: TaskOperation = new TaskOperation();
+
+        let experiment: IExperiment;
+
+        if (filePath.endsWith('.json')) {
+            let rawdata;
+            try {
+                rawdata = fs.readFileSync(filePath);
+            } catch (err) {
+                throw tl.loc("FileNotFound", err);
```

Comment:
Though "FileNotFound" is a common cause of the failure, though failure could be because of other exceptions too. So, can we have a generic message is the loc string and let err msg should print the exact error.